xml/admin_nfsganesha.xml

```diff
@@ -39,403 +28,148 @@
+     Execute the following to create a <literal>nfs-ganesha</literal> service:
+   </para>
 <screen>
-NFS_CORE_PARAM
-{
-NFS_Port = 2060;
-RQUOTA_Port = 876;
-}
+&prompt.cephuser;ceph orch apply nfs <replaceable>SERVICE_NAME</replaceable> --pool <replaceable>POOL_NAME</replaceable> --namespace <replaceable>NAMESPACE_NAME</replaceable>
 </screen>
-</sect2>
+ </sect1>
 
-  <sect2 xml:id="ceph-nfsganesha-config-exports-general">
-   <title>Exports Configuration</title>
+ <sect1 xml:id="ceph-nfsganesha-list-objects">
+   <title>List Objects in the NFS Recovery Pool</title>
    <para>
-    Exports configuration is stored as &rados; objects in the &ceph; cluster.
-    Each export block is stored in its own &rados; object named
-    <literal>export-<replaceable>ID</replaceable></literal>, where
-    <replaceable>ID</replaceable> must match the <literal>Export_ID</literal>
-    attribute of the export configuration. The association between exports and
-    &ganesha; services is done through the
-    <literal>conf-<replaceable>MINION_ID</replaceable></literal> objects. Each
-    service object contains a list of RADOS URLs for each export exported by
-    that service. An export block looks like the following:
+     Execute the following to list the objects in the NFS recovery pool:
    </para>
-<screen>EXPORT
-{
-  Export_Id = 1;
-  Path = "/";
-  Pseudo = "/";
-  Access_Type = RW;
-  Squash = No_Root_Squash;
-  [...]
-  FSAL {
-    Name = CEPH;
-  }
-}</screen>
+<screen>&prompt.cephuser;rados --pool <replaceable>POOL_NAME</replaceable> --namespace <replaceable>NAMESPACE_NAME</replaceable> ls</screen>
+ </sect1>
+
+ <sect1 xml:id="ceph-nfsganesha-create-export">
+   <title>Create an NFS Export</title>
    <para>
-    To create the &rados; object for the above export block, we first need to
-    store the export block code in a file. Then we can use the &rados; CLI tool
-    to store the contents of the previously saved file in a &rados; object.
+     Execute the following to create an NFS export.
    </para>
-<screen>
-&prompt.cephuser;rados -p <replaceable>POOL</replaceable> -N <replaceable>NAMESPACE</replaceable> put export-<replaceable>EXPORT_ID</replaceable> <replaceable>EXPORT_FILE</replaceable>
-</screen>
+   <note>
+     <para>
+       The FSAL block should be modified to include the desired cephx
+       user ID and secret access key.
+     </para>
+   </note>
+<screen>&prompt.cephuser;rados --pool <replaceable>POOL_NAME</replaceable> --namespace <replaceable>NAMESPACE_NAME</replaceable> put <replaceable>export-1</replaceable> <replaceable>export-1</replaceable></screen>
    <para>
-    After creating the export object, we can associate the export with a
-    service instance by adding the corresponding &rados; URL of the export
-    object to the service object. The following sections describe how to
-    configure an export block.
+     An update may be required for the configured exports to be recognized by
+     the &ganesha; service:
```

Comment:
resolved upstream by: https://review.gerrithub.io/c/ffilz/nfs-ganesha/+/482693

a restart should no longer be requried.